### DISCUSSION TITLE
Resources for translators

### DISCUSSION THREAD

**Discussion by allie-tatarian** [WeblateOrg/weblate#5771] [Q&A]
Hello all, I am a librarian working on a project called the [Library Space Technology Network](https://hosted.weblate.org/projects/library-space-technology-network/), and we are anticipating a wide variety of translators with different levels of technical background. I am working on making materials for those in our community that are new to Weblate and crowdsourced translation tools, and was wondering what sorts of resources people already point their translators towards. I will point our users to the [Translating using Weblate](https://docs.weblate.org/en/latest/user/translating.html) page, but I believe that some aspects of the page may be confusing to casual users and those who do not have a background in programming. For example, some parts of the page seem to be directed towards administrators, not translators, and some of the programming jargon (like "strings") may be confusing. Is there another page for translators in the documentation that I am missing? Is there a blog post or something similar with a how-to or list of resources for translators? Is anyone else collecting and curating resources that I may have missed? Thanks in advance for your help!

**Comment by orangesunny** [comment]
Hi @allie-williams,

Welcome to the community!

Thanks for pointing this out. The part of the docs you linked is too complex; I have to agree. There is nothing like a *Translator’s quickstart guide* for the newcomers at this moment. Let’s write it.

There are two ways to do it:

1. You can contribute it yourself: https://docs.weblate.org/en/latest/contributing/index.html#contribute-to-weblate-documentation.
2. You can share your collected information, and I will polish it to the docs article. This results in us being co-authors.

**Reply by allie-tatarian** [reply]
Thank you for the response! I'm glad to hear that we didn't miss any resources, that's reassuring. I am working to make a tutorial right now but since our project isn't using all of Weblate's functionalities right now, it will be a little pared-down, I think (for example we aren't using the glossary or plurals). I should have a draft in a few weeks and would love to share it for some polishing up; what would be the best way to share with you?

**Reply by nijel** [reply]
If you can do a pull request, that's probably the easiest approach - you can allow editing from the maintainers and together the documentation can be polished before it gets merged.

In case it's too technical for you, feel free to post it here for a review.

**Reply by allie-tatarian** [reply]
Hi again, I think I wasn't entirely clear about what I meant here. The issue isn't that I don't know how to make a pull request (I do!), it's that I'm creating resources for my community that aren't presented in the same way that the Weblate docs are. Because of this, I didn't think adding my writing to the Weblate translator's guide and making a pull request was the best way to share. If you're interested, I've put my tutorial on [the LSTN website](https://lstn.wolba.ch/weblate/).

**Reply by nijel** [reply]
I think something like this would be a great addition to our user docs. Our docs currently more describe how things works, but having these howtos would be better. I think it could be addded in https://docs.weblate.org/en/latest/user/basic.html.

@orangesunny any opinion on that?

**Reply by nijel** [reply]
Created https://github.com/WeblateOrg/weblate/issues/6169 to get something like this to the docs.

**Comment by comradekingu** [comment]
@allie-tatarian I had a go in https://github.com/WeblateOrg/weblate/issues/6169#issuecomment-860242734
